spec/models/manageiq/providers/vmware/infra_manager/provision_workflow_spec.rb

```diff
@@ -88,7 +88,22 @@
       workflow.instance_variable_set(:@target_resource, nil)
     end
 
-    context 'vlans' do
+    context '#set_on_vm_id_changed' do
+      it 'clears StorageProfile filter' do
+        workflow.instance_variable_set(
+          :@filters,
+          :Host            => {21  => "Platform / ESX 6.0"},
+          'StorageProfile' => {1   => "Tag 1"},
+          :src_vm_id       => @src_vm.id
```

Comment:
I did not recall that the other values are stored as symbols.  `StorageProfile` should be a symbol for consistency.  Also `:src_vm_id` is not stored in this hash so I it should not be in this hash.